src/applications/appeals/testing/sc/pages/housingRisk.js

```diff
@@ -15,6 +17,11 @@ export default {
         Y: 'Yes',
         N: 'No',
       },
+      updateUiSchema: () => ({
+        'ui:options': {
+          labelHeaderLevel: isOnReviewPage ? 4 : 3,
```

Comment:
I think `isOnReviewPage` is a function? Should this be `isOnReviewPage() ? 4 : 3`?
(same comment applies to `livingSituation.js`, `optionForMst.js` and `optionIndicator.js`)

Reply:
Derp, good catch!